src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -307,7 +309,7 @@ private static DetailAST getBraceFromSwitchMember(DetailAST ast) {
      * @return {@code DetailAST} if the first child is {@code TokenTypes.SLIST},
      *     {@code null} otherwise.
      */
-    private static DetailAST getBraceAsFirstChild(DetailAST ast) {
+    private static @Nullable DetailAST getBraceAsFirstChild(DetailAST ast) {
```

Comment:
Please make annotation above method definition. We should also make a check module for this. As we grow in annotation usage, we will not want to inline all of them.

Reply:
https://github.com/checkstyle/checkstyle/issues/14101#issue-2026222144

Reply:
@romani assigned you for approval ^

Reply:
approved, I surprised that it was not done from the beginning.